src/components/Navbar.jsx

```diff
@@ -41,6 +41,11 @@ const Navbar = () => {
           </NavLink>
         </Menu.Item>
         {renderCategories}
+        <Menu.Item>
+          <NavLink to="/subscription" id='subscription-link'>
```

Comment:
when should a user see this link?

Reply:
When logged in, I changed it back to [WIP] for this